modules/ROOT/pages/idt.adoc

```diff
@@ -0,0 +1,200 @@
+= Interrupt Descriptor Table
+:description: 
+:keywords: interrupts, x86
+:page-category: interrupts
+:source-language: c
+:table-caption!:
+
+== Overview
+
+The *Interrupt Descriptor Table* is a binary data structure specific to xref:x86.adoc[x86]'s 32-bit (IA-32) and 64-bit (x86-64) variants. It is used in xref:x86.adoc#_protected_mode_32_bit[*Protected Mode*] and xref:x86.adoc#_long_mode_x86_64_only[*Long Mode*] and a similar *Interrupt Vector Table* exists in xref:x86.adoc#_real_mode[*Real Mode*].
+
+The *IDT* is responsible for informing the CPU of the location and operation modes of *Interrupt Service Routines* corresponding to one of 256 *Interrupt Vectors*.
+
+The *IDT* is similar to the xref:gdt.adoc[*Global Descriptor Table*] in structure and interfaces with it, making it important to have a working and complete *GDT* before it can be used.
+
+== Interrupt Descriptor Table Register (IDTR)
+
+The location and size of the *Interrupt Descriptor Table* is held in the *Interrupt Descriptor Table Register* (*IDTR*). It is formatted as follows
+
+.*IDTR*
+[cols="10,1,5,1", grid=rows, stripes=odd]
+|===
+|47 (32-bit Mode) +
+79 (64-bit Mode)
+|16
+|15
+|0
+
+|*Offset* +
+31 (32-bit Mode) +
+63 (64-bit Mode)
+|{zwsp} +
+0
+|*Size* +
+15
+|{zwsp} +
+0
+|===
+
+* *Size:* The size of the *IDT* in bytes subtracted by 1.
+* *Offset:* The virtual address of the *IDT*.
```

Comment:
Format this using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/).